app/models/stripe_card.rb

```diff
@@ -387,8 +387,8 @@ def expired?
     Time.now.utc > Time.new(stripe_exp_year, stripe_exp_month).end_of_month
   end
 
-  def ephemeral_key(nonce:)
-    Stripe::EphemeralKey.create({ nonce:, issuing_card: stripe_id }, { stripe_version: "2020-03-02" })
+  def ephemeral_key(nonce:, stripe_version: "2020-03-02")
```

Comment:
by default, should we not pass in `nil`?

Reply:
We have an app-wide Stripe version configured in `Partners::Stripe::Shared::Opts`